app/views/bulk_update/confirmations/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render PageTitle::View.new(i18n_key: "bulk_placements.confirmation.show") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <div class="govuk-panel govuk-panel--confirmation trn-submission-success-panel govuk-!-margin-bottom-6">
+      <h1 class="govuk-panel__title">
+        Placement data submitted
+      </h1>
+    </div>
```

Comment:
Can we use 

https://govuk-components.netlify.app/components/panel/#panel-with-a-title-and-one-line-of-text

```suggestion

    <%= govuk_panel(title_text: "Placement data submitted")             %>
```

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/pull/3793/commits/84e29ed17a4ae57cadb7054c2a7adeac70fb115a